CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # MicroProfile Language Server Changelog
 
+## [0.7.0](https://github.com/eclipse/lsp4mp/milestone/7?closed=1) (March 28, 2023)
+
+### Enhancements
+ * Validation for GraphQL `@Query` and `@Mutation` methods with `void` return type. See [#348](https://github.com/eclipse/lsp4mp/issues/348), [#359](https://github.com/eclipse/lsp4mp/issues/359).
+ * Navigate to REST endpoints using WorkspaceSymbols. See [#87](https://github.com/eclipse/lsp4mp/issues/87).
+ * Show config property documentation when hovering over the key in a properties file. See [#321](https://github.com/eclipse/lsp4mp/issues/321).
+ * Validate lists in `@ConfigProperty`'s `defaultValue`. See [#351](https://github.com/eclipse/lsp4mp/pull/351).
+
+### Bug Fixes
+ * Hover fails in properties files when the Java language server is loading. See [#375](https://github.com/eclipse/lsp4mp/issues/375).
+ * Definition sometimes fails on property values in a properties file. See [#374](https://github.com/eclipse/lsp4mp/issues/374).
+ * Only show Java file snippets relevant to the context. See [#108](https://github.com/eclipse/lsp4mp/issues/108).
+ * Adjust go to definition range for property keys to include the offset between the property key and `=`. See [#335](https://github.com/eclipse/lsp4mp/pull/335).
+ * Fix NullPointerException during go to definition in properties files. See [#372](https://github.com/eclipse/lsp4mp/issues/372).
+ * Fix NullPointerException on shutdown when LSP client doesn't define extendedClientCapabilities. See [#363](https://github.com/eclipse/lsp4mp/pull/363).
+ * Completion causes Exceptions when typing in a Java file. See [#347](https://github.com/eclipse/lsp4mp/issues/347).
+ * Support projects that use classes from the `jakarta` namespace (JakartaEE 9). See [#344](https://github.com/eclipse/lsp4mp/issues/344).
```

Comment:
What about https://github.com/eclipse/lsp4mp/pull/332, which closes : https://github.com/eclipse/lsp4mp/issues/229, https://github.com/eclipse/lsp4mp/issues/320 ? If they should be mentioned, does it make sense to turn this entry into some general "Support the Jakarta namespace" ?